fr.openfisca.org/fr.openfisca.org.conf

```diff
@@ -68,11 +68,9 @@ server {
         default_type application/json;
     }
 
-    location ~/legislation(.*)$ {
-        proxy_pass http://legislation-explorer$1;
-        proxy_set_header Host $http_host;
-        proxy_http_version 1.1;
-        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+    location /legislation/ {
+        proxy_pass http://legislation-explorer$request_uri;
+        include proxy_params;
```

Comment:
Where are those `proxy_params` defined? 🤔
Unless I am mistaken, the [`include`](https://nginx.org/en/docs/ngx_core_module.html#include) Nginx directive targets files, and I don't see a `proxy_params` file in the codebase.

Reply:
This references `/etc/nginx/proxy_params`, which is shipped with the Debian package of Nginx. I think that it is an Nginx idiom, however I can add a comment to make this explicit.